who_are_we.md

```diff
@@ -0,0 +1,111 @@
+
+They kept the name "Gizra" which means pattern or silhouette in Hebrew, and posted in community related sites that there's a new kid in town.
+Thanks to their contributions, people in the community already knew their names, so the very next day they already got their first $100 work order. Gizra was on its way to the top!
+
+## Technical Recognition
+
+Like any new company, we struggled to get clients (a struggle we still face, although we're already equipped with better tools and knowledge). Even though income was poor, Brice and Amitai decided to travel to the 2009 DrupalCon Paris.
+
+It was the first time they attended a conference, let alone an international one, and it was a blast. They finally got to meet their Drupal heroes, people that at certain periods they were even in daily contact with them, thanks to their contributions and involvement.
+
+Finally, Brice and Amitai had the opportunity to meet in person people like [fago](https://www.drupal.org/u/fago), [klausi](https://www.drupal.org/u/klausi) and [dasjo](https://www.drupal.org/u/dasjo), that they were working with on the issue queue. Their first dinner with their new yet old Drupal friends, was in fact a first in what to become many more dinners and DrupalCamps, DrupalCons and other social events. In Paris they got their first taste of the Drupal community, and have been loving it ever since.
+
+Beyond that, Brice and Amitai were fortunate enough to meet [Moshe Weitzman](https://www.drupal.org/u/moshe-weitzman), a legendary Drupal core contributor who also had many high profile modules (Drush, Organic groups and others). Moshe knew Amitai from the issue queue, where Amitai was submitting patches to either fix bugs (and most probably introducing new bugs in the process), or suggest new features. As it so happened, Moshe was already looking for a new Organic Groups (OG) module maintainer. OG is a high profile module, that exists forever and is responsible for mini-site functionality. That is, on a single site, you can spin up as many subsites as you want, and each one can behave completely independently from each other, while still giving the site admin an easy way to administer them all.
+
+Moshe suggested that Amitai take ownership of OG, to which he immediately replied "no." It was a huge responsibility and he felt completely unworthy.  Thirty minutes later, he said "yes, but you should stick around".  
+The next day Amitai already suggested to Moshe a complete re-write of Organic groups.
+
+They returned back to Israel, with commit access to one of the most important Drupal modules. Even back then, they knew this was an important milestone.
+
+
+## Theatrical Recognition
+
+Even though we were handed a significant module, Gizra was still quite unknown by the international community. We did however hire our first employees: an office manager who was also a project manager, and followed by a developer.
```

Comment:
@amitaibu Is this correct? I was not sure if it was two employees that you hired.